engine/src/main/java/org/hibernate/search/query/dsl/sort/impl/SortFieldStates.java

```diff
@@ -43,7 +43,7 @@
 		initScalarMinMax( SortField.Type.INT, Integer.MIN_VALUE, Integer.MAX_VALUE );
 	}
 
-	private static void initScalarMinMax(Type type, double minValue, double maxValue) {
+	private static void initScalarMinMax(Type type, Object minValue, Object maxValue) {
```

Comment:
No need to change it, but this could be a `Number` instead of `Object`, isn't it?